.vscode/settings.json

```diff
@@ -41,5 +41,8 @@
     "editor.codeActionsOnSave": {
       "source.fixAll.stylelint": true
     }
-  }
+  },
+  "python.analysis.extraPaths": [
```

Comment:
I understand it may improve your local vscode setup, is it okay to undo this change? Thanks.